cli_reference/get_started_cli.adoc

```diff
@@ -188,6 +188,16 @@ Customer Portal] as a *_tar.gz_* archive. After logging in with your Red Hat
 account, you must have an active OpenShift Enterprise subscription to access the
 downloads page.
 
+[NOTE]
+===
+For other distributions of Linux, it is recommended to ensure that *libc* is included with
+your distribution. Attempting to use the CLI without libc could result in runtime errors such as:
```

Comment:
Additionally, it may be in the distribution, but not on the libpath by default ... some sort of reminder to check your libpath would also be good here IMO

Reply:
I think it would be better to point out the distro restriction in the note itself. Maybe "If you do not use RHEL or Fedora, ensure that *libc* is installed and on your libpath. If *libc* is not available, you might see the following error when you run CLI commands:"